### PR TITLE
Added a few icons I liked, and some from HowToMen

### PR DESCRIPTION
**Pull request by RikKoedoot**
I've added the following apps:

- OneDrive
- Microsoft Teams
- Rabobank
- Karwei
- Mijn HR
- Authy
- Pushbullet
- Slack
- Boom

### CODE REVIEW

File: svgs/onedrive.svg [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  </g>
+  <path
+     style="fill:#000002;fill-opacity:1;stroke:none;stroke-width:1.25102px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+     d="m 21.463916,16.00207 0.01982,-0.409078 -6.081788,-4.864468 1.064289,5.459668 z"
+     id="path3502" />
+</svg>
```
Comment (patrykmichalik) [inline svgs/onedrive.svg:94]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   viewBox="0 0 24 24"
   width="24"
   height="24"
   version="1.1"
   id="svg10"
   sodipodi:docname="onedrive.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs14">
	
	
	
	
</defs>
  <sodipodi:namedview
     id="namedview12"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     showguides="true"
     inkscape:guide-bbox="true"
     inkscape:zoom="22.627417"
     inkscape:cx="3.1819805"
     inkscape:cy="16.197165"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg10"
     scale-x="0.02"
     units="px">
    <sodipodi:guide
       position="2,24.701501"
       orientation="-1,0"
       id="guide16"
       inkscape:label=""
       inkscape:locked="false"
       inkscape:color="rgb(0,0,255)" />
    <sodipodi:guide
       position="7.4211366,22"
       orientation="0,1"
       id="guide18"
       inkscape:label=""
       inkscape:locked="false"
       inkscape:color="rgb(0,0,255)" />
    <sodipodi:guide
       position="11.224008,2"
       orientation="0,1"
       id="guide22"
       inkscape:label=""
       inkscape:locked="false"
       inkscape:color="rgb(0,0,255)" />
    <sodipodi:guide
       position="20,11.96875"
       orientation="0,1"
       id="guide24"
       inkscape:label=""
       inkscape:locked="false"
       inkscape:color="rgb(0,0,255)" />
    <sodipodi:guide
       position="22,17.354206"
       orientation="-1,0"
       id="guide45"
       inkscape:label=""
       inkscape:locked="false"
       inkscape:color="rgb(0,0,255)" />
    <sodipodi:guide
       position="0.16347331,0.43464525"
       orientation="-1,0"
       id="guide162"
       inkscape:label=""
       inkscape:locked="false"
       inkscape:color="rgb(0,0,255)" />
  </sodipodi:namedview>
  <path
     fill="#0364b8"
     d="M 17.747858,10.419838 C 16.954534,7.3264593 13.803745,5.4618854 10.710349,6.2552094 9.3173069,6.6124603 8.1073815,7.4760203 7.316734,8.6773053"
     id="path11234"
     style="fill:none;stroke:#000000;stroke-width:1.5;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
     sodipodi:nodetypes="ccc" />
  <path
     id="path11236"
     style="fill:none;stroke:#000000;stroke-width:1.5;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
     d="M 20.8032,15.951411 13.689386,11.690712 9.8063305,9.3652161 C 9.0765601,8.9132273 8.2344559,8.6738078 7.3760334,8.6749755 c -0.019831,0 -0.039832,0.00157 -0.059628,0.0018 -2.5533527,0.031636 -4.5976829,2.1283605 -4.5660679,4.6817135 0.011461,0.926214 0.301357,1.827711 0.8311798,2.587498 0.8639372,1.242134 2.2814807,1.981167 3.7945161,1.980375 H 17.494742"
     sodipodi:nodetypes="cccccccc" />
  <path
     id="path11238"
     style="fill:none;stroke:#000000;stroke-width:1.5;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
     d="m 3.5815173,15.94599 10.1078687,-4.255279 2.5498,-1.074685 c 0.235147,-0.10076 0.743193,-0.206919 1.255556,-0.206416 0.08536,0 0.169012,0.0052 0.252969,0.01084 0.535122,0.03623 1.056451,0.186528 1.528646,0.440886 1.827428,0.984363 2.511204,3.262645 1.526843,5.090072 -0.655727,1.216734 -1.926265,1.975351 -3.308458,1.974955 h -6.943984"
     sodipodi:nodetypes="cccccccc" />
</svg>
```

File: app/src/main/res/drawable/onedrive.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      android:fillColor="#000002"
+      android:strokeColor="#000000"/>
+  <path
+      android:pathData="m21.4639,16.0021 l0.0198,-0.4091 -6.0818,-4.8645 1.0643,5.4597z"
+      android:strokeLineJoin="miter"
+      android:strokeWidth="1.25102"
+      android:fillColor="#000002"
+      android:strokeColor="#00000000"
+      android:strokeLineCap="butt"/>
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/onedrive.xml:19]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="M17.7479,10.4198C16.9545,7.3265 13.8037,5.4619 10.7103,6.2552 9.3173,6.6125 8.1074,7.476 7.3167,8.6773"
      android:strokeWidth="1.5"
      android:fillColor="#00000000"
      android:strokeColor="#000000"/>
  <path
      android:pathData="M20.8032,15.9514 L13.6894,11.6907 9.8063,9.3652C9.0766,8.9132 8.2345,8.6738 7.376,8.675c-0.0198,0 -0.0398,0.0016 -0.0596,0.0018 -2.5534,0.0316 -4.5977,2.1284 -4.5661,4.6817 0.0115,0.9262 0.3014,1.8277 0.8312,2.5875 0.8639,1.2421 2.2815,1.9812 3.7945,1.9804H17.4947"
      android:strokeWidth="1.5"
      android:fillColor="#00000000"
      android:strokeColor="#000000"/>
  <path
      android:pathData="m3.5815,15.946 l10.1079,-4.2553 2.5498,-1.0747c0.2351,-0.1008 0.7432,-0.2069 1.2556,-0.2064 0.0854,0 0.169,0.0052 0.253,0.0108 0.5351,0.0362 1.0565,0.1865 1.5286,0.4409 1.8274,0.9844 2.5112,3.2626 1.5268,5.0901 -0.6557,1.2167 -1.9263,1.9754 -3.3085,1.975h-6.944"
      android:strokeWidth="1.5"
      android:fillColor="#00000000"
      android:strokeColor="#000000"/>
</vector>
```

File: svgs/onedrive.svg (hunk 1 above)
Comment (patrykmichalik) [inline svgs/onedrive.svg:94]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   viewBox="0 0 24 24"
   width="24"
   height="24"
   version="1.1"
   id="svg10"
   sodipodi:docname="onedrive.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs14">
	
	
	
	
</defs>
  <sodipodi:namedview
     id="namedview12"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     showguides="true"
     inkscape:guide-bbox="true"
     inkscape:zoom="22.627417"
     inkscape:cx="3.1819805"
     inkscape:cy="16.197165"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg10"
     scale-x="0.02"
     units="px">
    <sodipodi:guide
       position="2,24.701501"
       orientation="-1,0"
       id="guide16"
       inkscape:label=""
       inkscape:locked="false"
       inkscape:color="rgb(0,0,255)" />
    <sodipodi:guide
       position="7.4211366,22"
       orientation="0,1"
       id="guide18"
       inkscape:label=""
       inkscape:locked="false"
       inkscape:color="rgb(0,0,255)" />
    <sodipodi:guide
       position="11.224008,2"
       orientation="0,1"
       id="guide22"
       inkscape:label=""
       inkscape:locked="false"
       inkscape:color="rgb(0,0,255)" />
    <sodipodi:guide
       position="20,11.96875"
       orientation="0,1"
       id="guide24"
       inkscape:label=""
       inkscape:locked="false"
       inkscape:color="rgb(0,0,255)" />
    <sodipodi:guide
       position="22,17.354206"
       orientation="-1,0"
       id="guide45"
       inkscape:label=""
       inkscape:locked="false"
       inkscape:color="rgb(0,0,255)" />
    <sodipodi:guide
       position="0.16347331,0.43464525"
       orientation="-1,0"
       id="guide162"
       inkscape:label=""
       inkscape:locked="false"
       inkscape:color="rgb(0,0,255)" />
  </sodipodi:namedview>
  <path
     fill="#0364b8"
     d="M 17.747858,10.419838 C 16.954534,7.3264593 13.803745,5.4618854 10.710349,6.2552094 9.3173069,6.6124603 8.1073815,7.4760203 7.316734,8.6773053"
     id="path11234"
     style="fill:none;stroke:#000000;stroke-width:1.5;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
     sodipodi:nodetypes="ccc" />
  <path
     id="path11236"
     style="fill:none;stroke:#000000;stroke-width:1.5;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
     d="M 20.8032,15.951411 13.689386,11.690712 9.8063305,9.3652161 C 9.0765601,8.9132273 8.2344559,8.6738078 7.3760334,8.6749755 c -0.019831,0 -0.039832,0.00157 -0.059628,0.0018 -2.5533527,0.031636 -4.5976829,2.1283605 -4.5660679,4.6817135 0.011461,0.926214 0.301357,1.827711 0.8311798,2.587498 0.8639372,1.242134 2.2814807,1.981167 3.7945161,1.980375 H 17.494742"
     sodipodi:nodetypes="cccccccc" />
  <path
     id="path11238"
     style="fill:none;stroke:#000000;stroke-width:1.5;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
     d="m 3.5815173,15.94599 10.1078687,-4.255279 2.5498,-1.074685 c 0.235147,-0.10076 0.743193,-0.206919 1.255556,-0.206416 0.08536,0 0.169012,0.0052 0.252969,0.01084 0.535122,0.03623 1.056451,0.186528 1.528646,0.440886 1.827428,0.984363 2.511204,3.262645 1.526843,5.090072 -0.655727,1.216734 -1.926265,1.975351 -3.308458,1.974955 h -6.943984"
     sodipodi:nodetypes="cccccccc" />
</svg>
```

File: svgs/pushbullet.svg [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+       height="100%" />
+  </mask>
+  <path
+     d="m 77,189.6 c -16.733,0 -16.733,0 -16.733,-16.733 V 81 c 0,-16.733 0,-16.733 16.733,-16.733 h 3.334 c 16.733,0 16.733,0 16.733,16.733 v 91.867 c 0,16.733 0,16.733 -16.733,16.733 z m 44.041,0 c -5.699,0 -8.508,-2.809 -8.508,-8.508 V 72.774 c 0,-5.698 2.809,-8.507 8.508,-8.507 h 37.537 c 32.178,0 52.628,32.273 52.628,63.025 0,30.752 -20.628,62.308 -52.628,62.308 z"
+     fill="#ffffff"
+     mask="url(#b)"
+     id="path19"
+     style="display:inline;fill:#000000;fill-opacity:1"
+     transform="matrix(0.13250386,0,0,0.13250386,-5.9856102,-4.8191787)" />
+</svg>
```
Comment (patrykmichalik) [inline svgs/pushbullet.svg:74]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   width="24"
   height="24"
   viewBox="0 0 24 24"
   preserveAspectRatio="xMidYMid"
   version="1.1"
   id="svg21"
   sodipodi:docname="pushbullet.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns:xlink="http://www.w3.org/1999/xlink"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <sodipodi:namedview
     id="namedview23"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:snap-page="true"
     inkscape:zoom="29.831986"
     inkscape:cx="6.1008342"
     inkscape:cy="13.123498"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg21" />
  <defs
     id="defs8">
    <path
       id="a"
       d="M 256,128 C 256,198.692 198.692,256 128,256 57.308,256 0,198.692 0,128 0,57.308 57.308,0 128,0 198.692,0 256,57.308 256,128" />
    <linearGradient
       x1="0.085900001"
       y1="0.019540001"
       x2="0.77471"
       y2="0.73896003"
       id="c">
      <stop
         stop-color="#4CB36B"
         offset="0%"
         id="stop3" />
      <stop
         stop-color="#3EA16F"
         offset="100%"
         id="stop5" />
    </linearGradient>
  </defs>
  <mask
     id="b"
     fill="#ffffff">
    <use
       xlink:href="#a"
       id="use10"
       x="0"
       y="0"
       width="100%"
       height="100%" />
  </mask>
  <path
     d="m 77,189.6 c -16.733,0 -16.733,0 -16.733,-16.733 V 81 c 0,-16.733 0,-16.733 16.733,-16.733 h 3.334 c 16.733,0 16.733,0 16.733,16.733 v 91.867 c 0,16.733 0,16.733 -16.733,16.733 z m 44.041,0 c -5.699,0 -8.508,-2.809 -8.508,-8.508 V 72.774 c 0,-5.698 2.809,-8.507 8.508,-8.507 h 37.537 c 32.178,0 52.628,32.273 52.628,63.025 0,30.752 -20.628,62.308 -52.628,62.308 z"
     fill="#ffffff"
     mask="url(#b)"
     id="path19"
     style="display:inline;fill:#000000;fill-opacity:1"
     transform="matrix(0.11925347,0,0,0.11925347,-3.1870489,-3.1372603)" />
</svg>
```

Comment (patrykmichalik) [inline svgs/pushbullet.svg:74]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   width="24"
   height="24"
   viewBox="0 0 24 24"
   preserveAspectRatio="xMidYMid"
   version="1.1"
   id="svg21"
   sodipodi:docname="pushbullet.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns:xlink="http://www.w3.org/1999/xlink"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <sodipodi:namedview
     id="namedview23"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:snap-page="true"
     inkscape:zoom="29.831986"
     inkscape:cx="6.1008342"
     inkscape:cy="13.123498"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg21" />
  <defs
     id="defs8">
    <path
       id="a"
       d="M 256,128 C 256,198.692 198.692,256 128,256 57.308,256 0,198.692 0,128 0,57.308 57.308,0 128,0 198.692,0 256,57.308 256,128" />
    <linearGradient
       x1="0.085900001"
       y1="0.019540001"
       x2="0.77471"
       y2="0.73896003"
       id="c">
      <stop
         stop-color="#4CB36B"
         offset="0%"
         id="stop3" />
      <stop
         stop-color="#3EA16F"
         offset="100%"
         id="stop5" />
    </linearGradient>
  </defs>
  <mask
     id="b"
     fill="#ffffff">
    <use
       xlink:href="#a"
       id="use10"
       x="0"
       y="0"
       width="100%"
       height="100%" />
  </mask>
  <path
     d="m 77,189.6 c -16.733,0 -16.733,0 -16.733,-16.733 V 81 c 0,-16.733 0,-16.733 16.733,-16.733 h 3.334 c 16.733,0 16.733,0 16.733,16.733 v 91.867 c 0,16.733 0,16.733 -16.733,16.733 z m 44.041,0 c -5.699,0 -8.508,-2.809 -8.508,-8.508 V 72.774 c 0,-5.698 2.809,-8.507 8.508,-8.507 h 37.537 c 32.178,0 52.628,32.273 52.628,63.025 0,30.752 -20.628,62.308 -52.628,62.308 z"
     fill="#ffffff"
     mask="url(#b)"
     id="path19"
     style="display:inline;fill:#000000;fill-opacity:1"
     transform="matrix(0.11925347,0,0,0.11925347,-3.1870489,-3.1372603)" />
</svg>
```

File: app/src/main/res/drawable/pushbullet.xml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <group>
+    <clip-path
+        android:pathData="M27.9354,12.1413C27.9354,21.5083 20.3418,29.1018 10.9749,29.1018 1.6079,29.1018 -5.9856,21.5083 -5.9856,12.1413 -5.9856,2.7744 1.6079,-4.8192 10.9749,-4.8192 20.3418,-4.8192 27.9354,2.7744 27.9354,12.1413"/>
+    <path
+        android:pathData="m4.2172,20.3036c-2.2172,0 -2.2172,0 -2.2172,-2.2172L2,5.9136c0,-2.2172 0,-2.2172 2.2172,-2.2172l0.4418,0c2.2172,0 2.2172,0 2.2172,2.2172l0,12.1727c0,2.2172 0,2.2172 -2.2172,2.2172zM10.0528,20.3036c-0.7551,0 -1.1273,-0.3722 -1.1273,-1.1273L8.9254,4.8237c0,-0.755 0.3722,-1.1272 1.1273,-1.1272l4.9738,0c4.2637,0 6.9734,4.2763 6.9734,8.3511 0,4.0748 -2.7333,8.2561 -6.9734,8.2561z"
+        android:fillColor="#000000"/>
+  </group>
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/pushbullet.xml:14]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:aapt="http://schemas.android.com/aapt"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <group>
    <clip-path
        android:pathData="M27.3418,12.1272C27.3418,20.5575 20.5077,27.3916 12.0774,27.3916 3.6471,27.3916 -3.187,20.5575 -3.187,12.1272 -3.187,3.6969 3.6471,-3.1373 12.0774,-3.1373 20.5077,-3.1373 27.3418,3.6969 27.3418,12.1272"/>
    <path
        android:pathData="m5.9955,19.4732c-1.9955,0 -1.9955,0 -1.9955,-1.9955L4,6.5223c0,-1.9955 0,-1.9955 1.9955,-1.9955l0.3976,0c1.9955,0 1.9955,0 1.9955,1.9955l0,10.9555c0,1.9955 0,1.9955 -1.9955,1.9955zM11.2475,19.4732c-0.6796,0 -1.0146,-0.335 -1.0146,-1.0146L10.2329,5.5413c0,-0.6795 0.335,-1.0145 1.0146,-1.0145l4.4764,0c3.8373,0 6.2761,3.8487 6.2761,7.516 0,3.6673 -2.46,7.4304 -6.2761,7.4304z"
        android:fillColor="#000000"/>
  </group>
</vector>
```

File: svgs/pushbullet.svg (hunk 3 above)
Comment (patrykmichalik) [inline svgs/pushbullet.svg:74]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   width="24"
   height="24"
   viewBox="0 0 24 24"
   preserveAspectRatio="xMidYMid"
   version="1.1"
   id="svg21"
   sodipodi:docname="pushbullet.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns:xlink="http://www.w3.org/1999/xlink"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <sodipodi:namedview
     id="namedview23"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:snap-page="true"
     inkscape:zoom="29.831986"
     inkscape:cx="6.1008342"
     inkscape:cy="13.123498"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg21" />
  <defs
     id="defs8">
    <path
       id="a"
       d="M 256,128 C 256,198.692 198.692,256 128,256 57.308,256 0,198.692 0,128 0,57.308 57.308,0 128,0 198.692,0 256,57.308 256,128" />
    <linearGradient
       x1="0.085900001"
       y1="0.019540001"
       x2="0.77471"
       y2="0.73896003"
       id="c">
      <stop
         stop-color="#4CB36B"
         offset="0%"
         id="stop3" />
      <stop
         stop-color="#3EA16F"
         offset="100%"
         id="stop5" />
    </linearGradient>
  </defs>
  <mask
     id="b"
     fill="#ffffff">
    <use
       xlink:href="#a"
       id="use10"
       x="0"
       y="0"
       width="100%"
       height="100%" />
  </mask>
  <path
     d="m 77,189.6 c -16.733,0 -16.733,0 -16.733,-16.733 V 81 c 0,-16.733 0,-16.733 16.733,-16.733 h 3.334 c 16.733,0 16.733,0 16.733,16.733 v 91.867 c 0,16.733 0,16.733 -16.733,16.733 z m 44.041,0 c -5.699,0 -8.508,-2.809 -8.508,-8.508 V 72.774 c 0,-5.698 2.809,-8.507 8.508,-8.507 h 37.537 c 32.178,0 52.628,32.273 52.628,63.025 0,30.752 -20.628,62.308 -52.628,62.308 z"
     fill="#ffffff"
     mask="url(#b)"
     id="path19"
     style="display:inline;fill:#000000;fill-opacity:1"
     transform="matrix(0.11925347,0,0,0.11925347,-3.1870489,-3.1372603)" />
</svg>
```

Comment (patrykmichalik) [inline svgs/pushbullet.svg:74]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   width="24"
   height="24"
   viewBox="0 0 24 24"
   preserveAspectRatio="xMidYMid"
   version="1.1"
   id="svg21"
   sodipodi:docname="pushbullet.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20, custom)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns:xlink="http://www.w3.org/1999/xlink"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <sodipodi:namedview
     id="namedview23"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:snap-page="true"
     inkscape:zoom="29.831986"
     inkscape:cx="6.1008342"
     inkscape:cy="13.123498"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="svg21" />
  <defs
     id="defs8">
    <path
       id="a"
       d="M 256,128 C 256,198.692 198.692,256 128,256 57.308,256 0,198.692 0,128 0,57.308 57.308,0 128,0 198.692,0 256,57.308 256,128" />
  </defs>
  <mask
     id="b"
     fill="#ffffff">
    <use
       xlink:href="#a"
       id="use10"
       x="0"
       y="0"
       width="100%"
       height="100%" />
  </mask>
  <path
     d="m 77,189.6 c -16.733,0 -16.733,0 -16.733,-16.733 V 81 c 0,-16.733 0,-16.733 16.733,-16.733 h 3.334 c 16.733,0 16.733,0 16.733,16.733 v 91.867 c 0,16.733 0,16.733 -16.733,16.733 z m 44.041,0 c -5.699,0 -8.508,-2.809 -8.508,-8.508 V 72.774 c 0,-5.698 2.809,-8.507 8.508,-8.507 h 37.537 c 32.178,0 52.628,32.273 52.628,63.025 0,30.752 -20.628,62.308 -52.628,62.308 z"
     fill="#ffffff"
     mask="url(#b)"
     id="path19"
     style="display:inline;fill:#000000;fill-opacity:1"
     transform="matrix(0.11925347,0,0,0.11925347,-3.1870489,-3.1372603)" />
</svg>
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Here are some tweaks I’d suggest.

**Review by RikKoedoot — COMMENTED** [review]
Yes

**Review by patrykmichalik — APPROVED** [review]

**Pull request merged**